### DISCUSSION TITLE
配車ボタンを押すと、一瞬アプリ画面が表示されて、すぐに「400 Bad Request」となるのですが、原因何か可能性ありますでしょうか？

### DISCUSSION THREAD

**Discussion by susumun** [dzeyelid/onsen-maas-vueapp#22] [Q&A] [answered]
配車ボタンを押すと、一瞬アプリ画面が表示されて、すぐに「400 Bad Request」となるのですが、原因何か可能性ありますでしょうか？

**Comment by susumun** [comment]
下記のエラーがコンソールで確認できました。
もう一度値を確認してみます。

エラー：Liff IDを取得できません。Error: LIFF app 'xxxxxxx' was not found.

**Comment by himanago** [comment, marked as answer]
こちらの、GitHub Actions の Secret 設定は正しく行われ、正常にデプロイされていますでしょうか？
https://github.com/dzeyelid/onsen-maas-vueapp/blob/main/docs/self-paced-handson/update-workflow.md

**Reply by susumun** [reply]
ありがとうございます！解決しました

**Reply by susumun** [reply]
LIFF IDにLIFF Login Channel IDを設定してしまっていました。

**Comment by k-miyake** [comment]
ここの手順を再度確認してみてください。
https://github.com/dzeyelid/onsen-maas-vueapp/blob/main/docs/self-paced-handson/update-workflow.md#%E3%83%AF%E3%83%BC%E3%82%AF%E3%83%95%E3%83%AD%E3%83%BC%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E3%82%92%E6%9B%B4%E6%96%B0%E3%81%99%E3%82%8B

**Reply by susumun** [reply]
ありがとうございました！
